packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js

```diff
@@ -74,7 +74,8 @@ export const getMenu = ({ DBAAS_LOGS_URL }) => [
           state: 'pci.projects.project.loadbalancer',
         },
         title: 'Load Balancer',
-        regions: ['EU', 'CA'],
+        regions: ['EU', 'CA', 'US'],
```

Comment:
Still needed with feature flipping ?

Reply:
By default, feature flipping is used. The regions mentioned here do not count when feature flipping is enabled. I just added them to avoid confusion when someone looks into it.

Reply:
Yes. I will remove them.